test/hotspot/jtreg/compiler/c2/irTests/TestSkeletonPredicates.java

```diff
@@ -29,8 +29,8 @@
 
 /*
  * @test
- * @bug 8278228
- * @summary C2: Improve identical back-to-back if elimination
+ * @bug 8282592
+ * @summary C2: assert(false) failed: graph should be schedulable
```

Comment:
Are the changes to this file intended? They seem to be unrelated.

Reply:
They are unrelated but I noticed, while working on this change, that that test has the wrong bug/summary. So yes unrelated but also intended (not sure a separate change is required?)